official/SpecificaTecnica/res/sections/designPatterns.tex

```diff
@@ -101,7 +101,7 @@ \subsubsection{Facade}
 \caption{Design pattern Facade}
 \end{figure}
 \paragraph{Contestualizzazione}\mbox{} \\
-È opinione del team che i vantaggi del design pattern Facade superino i suoi svantaggi. Essendo MaaS un prodotto complesso, con molti sottosistemi che interagiscono tra loro, l'utilizzo del Facade permette di semplificare la loro gestione e le loro interazioni. Inoltre l'accoppiamento dei sottosistemi viene ridotto, favorendo quindi il riuso dei singoli componenti dell'applicazione. Gli aspetti negativi non possono tuttavia essere sottovalutati: il Facade rischia di diventare un \textit{Single point of failure}; di conseguenza sarà fatta molta attenzione a verificarne il funzionamento attraverso casi di test creati ad hoc.
+È opinione del \glossaryItem{team} che i vantaggi del design pattern Facade superino i suoi svantaggi. Essendo \glossaryItem{MaaS} un prodotto complesso, con molti sottosistemi che interagiscono tra loro, l'utilizzo del Facade permette di semplificare la loro gestione e le loro interazioni. Inoltre l'accoppiamento dei sottosistemi viene ridotto, favorendo quindi il \glossaryItem{riuso} dei singoli componenti dell'applicazione. Gli aspetti negativi non possono tuttavia essere sottovalutati: il Facade rischia di diventare un \textit{Single point of failure}; di conseguenza sarà fatta molta attenzione a verificarne il funzionamento attraverso \glossaryItem{casi} di test creati ad hoc.
```

Comment:
Leggendo a caso la pr, se questo è un errore allora è presente anche nelle Norme di Progetto di cui ho già eseguito il merge (issue#447). In caso è da correggere